pkg/net/ip.go

```diff
@@ -112,6 +142,14 @@ func IP(ip cue.Value) bool {
 	return netGetIP(ip) != nil
 }
 
+// IPCIDR reports whether ip is a valid IPv4 or IPv6 address with CIDR subnet notation.
+//
+// The address may be a string or list of bytes.
+func IPCIDR(ip cue.Value) bool {
+	// TODO: convert to native CUE.
+	return netGetIPCIDR(ip) != nil
```

Comment:
CUE can pass error messages back to the user if you use the `IPCIDR(ip cue.Value) (bool, error)` signature. I know it looks strange, but I would do that here for the sake of better error messages.